log/log.go

```diff
@@ -8,16 +8,27 @@ import (
 
 // Logger is the interface used to log panics that occur during query execution. It is settable via graphql.ParseSchema
 type Logger interface {
-	LogPanic(ctx context.Context, value interface{})
+	LogPanic(ctx context.Context, value interface{}, info interface{})
```

Comment:
I don't want the Logger interface to add one more argument to this method. This is a breaking change of the API and I want to avoid it as much as possible.